src/App.tsx

```diff
@@ -1,20 +1,20 @@
 import * as React from 'react';
-import './App.css';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
-const logo = require('./logo.svg');
+import Header from './components/Header/Header';
```

Comment:
I like the organization this gives personally. We'll have more than just this one file per component in most situations. At minimum there will be a test file alongside the component, as well as usually a `.css` file to go with. I feel like we'd quickly bloat the components folder without more organization.